### PR TITLE
Update Readme according to Repo migration

### PR DESCRIPTION
**Pull request by el00ruobuob**
Add instruction to contribute on https://repo.getmonero.org/monero-project/monero-site as the repository has been moved.
Please review @moneromooo-monero & @fluffypony

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Repository moved to https://repo.getmonero.org/monero-project/monero-site
+
+This repository has recently been moved to https://repo.getmonero.org/monero-project/monero-site  
+Please do your changes on the new repository.
```
Comment (erciccione) [inline README.md]: maybe use `submit` instead of `do`?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
+To contribute on https://repo.getmonero.org/ please proceed this way:
+
+* Login using OAuth2 with your GitHub account
```
Comment (erciccione) [inline README.md]: Don't think that using a GitHub account is necessary, people can just subscribe to GitLab

File: README.md (hunk 1 above)
Comment (el00ruobuob) [reply README.md]: Or course yes

File: README.md (hunk 2 above)
Comment (el00ruobuob) [reply README.md]: I'll write both options then.

**Comment by el00ruobuob** [comment]
Corrections done @erciccione

**Pull request closed without merging**